public/app/plugins/panel/bargauge/BarGaugePanel.tsx

```diff
@@ -69,6 +70,8 @@ export class BarGaugePanel extends PureComponent<PanelProps<BarGaugeOptions>> {
 
   getValues = (): FieldDisplay[] => {
     const { data, options, replaceVariables, fieldConfig, timeZone } = this.props;
+    ensureGlobalRangeOnState(data.series);
```

Comment:
We will need this in Stat & Gauge as well. I can add that

Reply:
hm.. since this is needed in all stat panels wonder if it's better to move this to getFieldDisplayValues